### PR TITLE
lab configuration apis

### PR DESCRIPTION
**Pull request by Aakriti28**

### CODE REVIEW

File: rest/models.py [hunk 1]
```diff
@@ -1,3 +1,23 @@
 from django.db import models
 
 # Create your models here.
+
+class MachineType(models.Model) :
```
Comment (nileshbhosale) [inline rest/models.py:5]: Please create different model.py files inside the models package for clarity. e.g. rest/models/machine_type.py

File: rest/serializers.py [hunk 2]
```diff
@@ -0,0 +1,17 @@
+from rest_framework_json_api import serializers
+from .models import MachineType, TestKit, LabConfiguration
+
+class MachineTypeSerializer(serializers.ModelSerializer):
```
Comment (nileshbhosale) [inline rest/serializers.py:4]: Please nest this files as well.

File: rest/views.py [hunk 3]
```diff
@@ -1,3 +1,22 @@
...
+from rest_framework.decorators import api_view
+
+from django.http.response import JsonResponse
+from rest_framework.parsers import JSONParser
+
+from .models import MachineType, TestKit, LabConfiguration
+from .serializers import MachineTypeSerializer, TestKitSerializer, LabConfigurationSerializer
 
 # Create your views here.
+class MachineTypeViewset(viewsets.ModelViewSet):
```
Comment (nileshbhosale) [inline rest/views.py:12]: views folder here too.

File: rest/models.py [hunk 4]
```diff
@@ -1,3 +1,23 @@
...
+	name = models.CharField(max_length=60, blank=False, default='PCR machine')
+	no_of_wells = models.SmallIntegerField()
+	plate_x = models.SmallIntegerField(default=12)
```
Comment (vyasakanksha) [inline rest/models.py:8]: plate_x and plate_y are describing the x,y dimensions of the plate. The names dim_x and dim_x would make more sense here.

**Pull request merged**